resources/content/cattle-global.properties

```diff
@@ -37,7 +37,7 @@ cattle.home=
 process.retry_max_wait.millis=120000
 
 # Choose mysql, hsqldb, or h2
-db.cattle.database=h2
+db.cattle.database=mysql
```

Comment:
The design of cattle is such that you can download the `cattle.jar` and just run it with zero dependencies.  We have other components that make this assumption.  As such we can't default to `mysql` and `h2` must be the default.  When running cattle in development just add `-Ddb.cattle.database=mysql` to the Java command line.